### ISSUE TITLE
delete_by_query

### ISSUE THREAD

**Issue by yyrr001** [NLPchina/elasticsearch-sql#339]
hi,
   Since the "_delete_by_query" now come back to elastic version 5.0.0, the "DELETE" command will come back too?

**Comment by eliranmoyal** [comment]
Yes, I'm having some troubles with delete specific type but it seems working.
Will be released this weekend

**Comment by yyrr001** [comment]
nice!

**Comment by yyrr001** [comment]
oh, "_update_by_query" is now available in the version 5.0. 
Is there any plan supporting  "UPDATE" command ?

**Comment by eliranmoyal** [comment]
Not in the first version but later

**Comment by eliranmoyal** [comment]
sorry man, somehow I'm still getting the weirdest errors again
```
{"type":"class_cast_exception","reason":"org.elasticsearch.index.reindex.DeleteByQueryRequest cannot be cast to org.elasticsearch.index.reindex.DeleteByQueryRequest"}
```
it looks like it is still a different module in elasticsearch and the import is on wrong order. i'll try and open a request for that

**Comment by yyrr001** [comment]
it's ok. I thought "delete_by_query" api is already not a plugin in version5.0. But now it still looks like something wrong although it's provides as api.
I hope you can finally solve it : )

**Comment by eliranmoyal** [comment]
yeah turns out it is still a plugin but provided by default
I opened an issue about plugins using other plugins then ,I just commented on it now and they say they still not worked on it ..

**Comment by yyrr001** [comment]
oh, if it still be provided as a plugin, may be the problem can't be solved like version2.x.x...

**Comment by ffpeng90** [comment]
is there any progress on it?  i have met same problem on elasticsearch 5.2.0.
